R/calculate_stats.R

```diff
@@ -105,9 +109,10 @@ calculate_stats <- function(seasons = nflreadr::most_recent_season(),
   # we need those to identify things like fumbles depending on playtype or
   # first downs depending on playtype
   playstats <- load_playstats(seasons = seasons) %>%
```

Comment:
Need to pull seasons from custom pbp. So the seasons arg is actually ignored in that case

Reply:
Yeah, can probably warn/check somehow?

Reply:
Yeah I think I'd print a message when custom_pbp is TRUE. But not every time. One of those "once every 8 hours" messages

Reply:
Here is an example how to do that with cli https://github.com/nflverse/nflseedR/blob/a8f90520c45ecf5e6d2e8abd61a641613e622761/R/simulations.R#L186-L193

Reply:
actually as long as pbp is nonzero rows playstats should be nonzero rows

Reply:
Yeah I never really thought about validation of the seasons arg because load_pbp checks for valid seasons.

Reply:
i'd even skip the message about ignoring pbp and just check the custom pbp for the seasons, if any are missing it should warn